lms/djangoapps/teams/static/teams/templates/teams_tab.underscore

```diff
@@ -7,6 +7,7 @@
     </div>
 </div>
 <div class="teams-header"></div>
+<div class="teams-instructor-tools-bar"></div>
```

Comment:
Note that @frrrances is still trying to decide if this instructional bar behaves as it does in the wireframes right now, or if it is really a "tab bar" within the "edit team mode" (which would remain visible when switching to the "edit membership" view). I don't think we can understand where this HTML element belongs until we have that decision.